src/main/java/be/seeseemelk/mockbukkit/inventory/InventoryMock.java

```diff
@@ -80,12 +74,9 @@ public void assertTrueForNonNulls(@NotNull Predicate<ItemStack> condition)
 	 *
 	 * @param condition The condition to check for.
 	 */
-	public void assertTrueForSome(@NotNull Predicate<ItemStack> condition)
-	{
-		for (ItemStack item : items)
-		{
-			if (item != null && condition.test(item))
-			{
+	public void assertTrueForSome(@NotNull Predicate<ItemStack> condition) {
+		for (ItemStack item : items) {
+			if (item != null && condition.test(item)) {
```

Comment:
You broke the formatting here.

```suggestion
	public void assertTrueForSome(@NotNull Predicate<ItemStack> condition)
	{
		for (ItemStack item : items)
		{
			if (item != null && condition.test(item))
			{
```